src/main/client/api/index.ts

```diff
@@ -17,8 +19,14 @@ export function useClientApi() {
         return registeredApis[apiName] as ClientPlugin[K];
     }
 
+    async function getSync<K extends keyof ClientPlugin>(apiName: K, timeout = 30000): Promise<ClientPlugin[K]> {
```

Comment:
I think this should be called `getAsync`.

Asynchronous implies there's an `await` necessary to get the api.